src/components/shared/DividendLineChart.tsx

```diff
@@ -21,13 +21,15 @@ const KeyFiguresChart: React.FC<DividendsChartProps> = ({
   const { t } = useTranslation();
 
   const options = {
-    // colors: ['#4392F1', '#F6AE2D'],
     chart: {
       height: 350,
       type: 'line',
       toolbar: {
         show: false,
       },
+      zoom: {
+        enabled: false,
+      },
     },
     stroke: {
       width: [0, 4],
```

Comment:
The component and proptype name is still messed up (just noticed the other day). I can fix this in a separate PR or you can do it real quick in this one.

Reply:
I'll fix it rn